### ISSUE TITLE
Voice Over: "Read Text Attributes" does not work in editor

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#42556]
### Issue Type
Bug

### Description

1. Enable voice over
1. In the editor, try using VO+T (ctrl+option+T) to trigger "read text attributes" on some text

**Expected**
Reads out information about current focused text, such as decorations or problems. This is what happens in programs like text edits

**Actual**
Nothing read. "Space" shows up in VO window 

### VS Code Info

VS Code version: Code - Insiders 1.20.0-insider (8cac1acd97466afc75c663dc1626d8562a9a3f3d, 2018-01-30T17:19:56.812Z)
OS version: Darwin x64 17.3.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2200)|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (0.10GB free)|
|Process Argv|/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron|
|Screen Reader|yes|
|VM|0%|

</details>
Reproduces with extensions disabled

**Comment by isidorn** [comment]
I can reproduce this with Chrome. 
@mjbvz would you mind filling an issue in Chrome. Let me know if you are busy and I can also file it.

**Comment by isidorn** [comment]
We do not have an upstream issue, nobody else complained about this. Thus closing as will not fix
